# 某一时刻某个机器人所执行的所有技能
string robot_id          # 机器人标识
SkillInfo[] skill_list   # 该机器人在此“时间步”执行的技能序列
